frontend/www/js/omegaup/components/course/Details.vue

```diff
@@ -288,5 +289,8 @@ export default class CourseDetails extends Vue {
     }
     return time.formatDateTime(date);
   }
+  getAlias(): string {
+    return this.course.alias + '_' + headerPayload.currentUsername;
+  }
```

Comment:
Este es un buen candidato para ser una [Computed property](https://vuejs.org/v2/guide/computed.html#Computed-Properties)
```suggestion
  get aliasWithUsername(): string {
    return `${this.course.alias}_${headerPayload.currentUsername}`;
  }
```
Además, esa es una mejor forma de realizar [Interpolación en JavaScript](https://dmitripavlutin.com/string-interpolation-in-javascript/)

Aplica para los dos archivos